### ISSUE TITLE
Server crash in processes/_base.js

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-emails#666]
Logs found in a user's instance:
```
/usr/local/cozy/apps/emails/build/server/processes/_base.js:70
          results.push(cb(err, arg1, arg2, arg3, arg4));
                       ^
TypeError: undefined is not a function
    at /usr/local/cozy/apps/emails/build/server/processes/_base.js:70:24
    at /usr/local/cozy/apps/emails/node_modules/async/lib/async.js:700:17
    at /usr/local/cozy/apps/emails/build/server/processes/message_remove_by_account.js:49:18
    at /usr/local/cozy/apps/emails/build/server/processes/message_remove_by_account.js:78:16
    at /usr/local/cozy/apps/emails/node_modules/cozydb/lib/cozymodel.js:254:18
    at parseBody (/usr/local/cozy/apps/emails/node_modules/cozydb/node_modules/request-json-light/main.js:75:10)
    at IncomingMessage.<anonymous> (/usr/local/cozy/apps/emails/node_modules/cozydb/node_modules/request-json-light/main.js:109:14)
    at IncomingMessage.emit (events.js:117:20)
    at _stream_readable.js:944:16
    at process._tickCallback (node.js:442:13)
```

Don't know if it's related, but the user deleted one of its remote account, so every request to refresh it resulted in a 500 error:
```
Error: Lookup failed c7mb14773581lbz
    at ImapConnection.Connection._resTagged (/usr/local/cozy/apps/emails/node_modules/imap/lib/Connection.js:1463:11)
    at Parser.<anonymous> (/usr/local/cozy/apps/emails/node_modules/imap/lib/Connection.js:173:10)
    at Parser.emit (events.js:95:17)
    at Parser._resTagged (/usr/local/cozy/apps/emails/node_modules/imap/lib/Parser.js:174:10)
    at Parser._parse (/usr/local/cozy/apps/emails/node_modules/imap/lib/Parser.js:138:16)
    at Parser._tryread (/usr/local/cozy/apps/emails/node_modules/imap/lib/Parser.js:82:15)
    at CleartextStream.Parser._cbReadable (/usr/local/cozy/apps/emails/node_modules/imap/lib/Parser.js:53:12)
    at CleartextStream.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
    at _stream_readable.js:420:7
```

**Comment by clochix** [comment]
@poupotte why is this ticket “to merge”, I don't see any commit ?

**Comment by aenario** [comment]
It's in https://github.com/aenario/cozy-emails/commit/8a878de7316c07963acbcd84c800909526dc25a4#diff-2bbae0fd84148a22444f80119f33c6eaL49

Which is part of the current Indexer2 PR

**Comment by jsilvestre** [comment]
All hail to issue #666.

**Comment by poupotte** [comment]
This issue is fixed in new version (0.9.18)
